chair/role.html

```diff
@@ -12,7 +12,7 @@
 . <a href="/Guide/">Art of Consensus Guide</a>
 
 <h1>W3C Working/Interest Group Chair</h1>
-The W3C Director appoints (or re-appoints) a Chair (or more than one co-Chair) for every Working
+The W3C Team appoints (or re-appoints) a Chair (or more than one co-Chair) for every Working
```

Comment:
Consider referring to https://www.w3.org/2023/Process-20230612/#ref-for-team%E2%91%A2%E2%91%A6 or https://www.w3.org/2023/Process-20230612/#team-decision

Reply:
+1 on referring the undated version of the Process